docs/samples/tensorflow/tensorflow-canary.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: "serving.kubeflow.org/v1alpha1"
```

Comment:
Can we leave this in the canary samples?

Reply:
this is showing TF e-2-e if somebody lands on TF sample. The one in canary samples is using the fake model. I plan to extend and continue making the TF example complete from all different scenarios in future